ACPs/83-dynamic-multidimensional-fees/README.md

```diff
@@ -47,24 +48,40 @@ $$base \  fee = \sum_{i=0}^3 r_i \times u_i$$
 ### Dynamic scheme components
 
 Fee rates are updated in time, to allow a fee increase when network is getting congested. Each new block is a potential source of congestion, as its transactions carry complexity that each validator must process to verify and eventually accept the block. The more complexity carries a block, and the more rapidly blocks are produced, the higher the congestion.  
-We seek a scheme that rapidly increases the fees when blocks complexity goes above a defined threshold and that equally rapidly decreases the fees once complexity goes down (because blocks carry less/simpler transactions, or because they are produced more slowly).
 
-Suppose that a block $B$ is the current chain tip. $B$ has the following features:
+We seek a scheme that rapidly increases the fees when blocks complexity goes above a defined threshold and that equally rapidly decreases the fees once complexity goes down (because blocks carry less/simpler transactions, or because they are produced more slowly). We define the desired threshold as a *target complexity rate* $T_i$: we would want to process every second a block whose cumulated complexity is $T_i$. Any complexity more than that causes some congestion that we want to penalize via fees; any congestion below $T_i$ causes the network to be under utilized.
```

Comment:
This is not exactly true that we want the fees to equally increase/decrease when block complexity is above/below target. Assuming that's multiplicative, then we'd end up with the same problem of long term usage being above target as in EIP-1559 (https://ethresear.ch/t/make-eip-1559-more-like-an-amm-curve/9082).

Could we re-phrase the goal here?